pipeline/tests/database_test.py

```diff
@@ -28,6 +28,7 @@ def test_table_counts():
     test.assert_table_count("rideshare", 671938)
     test.assert_table_count("cta_train_ridership", 104564)
     test.assert_table_count("taxitrips", 100000)
+    test.assert_table_count("sidewalk_cafe", 19858)
```

Comment:
Thanks for adding this, this test will help others know if they have the correct version of the database.